app/src/main/res/raw/level1.json

```diff
@@ -0,0 +1,17 @@
+{
+  "baseSpeed": 50,
+  "timeline": [
+    {
+      "interval": 50,
+      "type": "cactus"
```

Comment:
do we have a cactus?

Reply:
Yes, it exists in mipmap resources, so I created a class for it